pkg/spec/job_test.go

```diff
@@ -127,3 +127,130 @@ func TestJobSpecMod_CreateKubernetesController(t *testing.T) {
 		})
 	}
 }
+
+func TestJobValidate(t *testing.T) {
+	tests := []struct {
```

Comment:
There should be at least one test cases that tests an invalid value.

Reply:
the one test case that is there is invalid one! It will fail!